lib/pages/new_group/new_group.dart

```diff
@@ -88,295 +30,17 @@ class NewGroupController extends ContactsSelectionController<NewGroup>
-  ) {
-    Logs().e(
-      'NewGroupController::_handleUploadAvatarNewGroupChatOnError() - error: $error | stackTrace: $stackTrace',
-    );
-  }
-
-  void _handleCreateNewGroupChatChatOnData(
-    BuildContext context,
-    Either<Failure, Success> event,
-  ) {
-    Logs().d('NewGroupController::_handleCreateNewGroupChatChatOnData()');
-    createRoomStateNotifier.value = event;
-    event.fold(
-      (failure) {
-        Logs().e(
-          'NewGroupController::_handleCreateNewGroupChatChatOnData() - failure: $failure',
-        );
-      },
-      (success) {
-        Logs().d(
-          'NewGroupController::_handleCreateNewGroupChatChatOnData() - success: $success',
-        );
-        if (success is CreateNewGroupChatSuccess) {
-          _goToRoom(success);
-        }
-      },
-    );
-  }
-
-  void _handleCreateNewGroupChatOnDone() {
-    Logs().d('NewGroupController::_handleCreateNewGroupChatOnDone() - done');
-  }
-
-  void _handleCreateNewGroupChatOnError(dynamic error, StackTrace? stackTrace) {
-    Logs().e(
-      'NewGroupController::_handleUploadAvatarNewGroupChatOnError() - error: $error | stackTrace: $stackTrace',
-    );
-  }
-
-  void _goToRoom(CreateNewGroupChatSuccess success) {
-    context.go(
-      "/rooms/${success.roomId}",
-      extra: ChatRouterInputArgument(
-        type: ChatRouterInputArgumentType.draft,
-        data: success.groupName,
-      ),
-    );
-  }
-
-  void _getImageOnWeb(
-    BuildContext context,
-  ) async {
-    final result = await FilePicker.platform.pickFiles(
-      type: FileType.image,
-    );
-    Logs().d(
-      'NewGroupController::_getImageOnWeb(): FilePickerResult - $result',
-    );
-    if (result == null || result.files.single.bytes == null) {
-      return;
-    } else {
-      avatarFilePickerNotifier.value = result;
-      Logs().d(
-        'NewGroupController::_getImageOnWeb(): AvatarWebNotifier - ${avatarFilePickerNotifier.value}',
+    if (responsiveUtils.isTwoColumnLayout(context)) {
+      Navigator.of(context).pushNamed(
+        'innernavigator/newgroupchatinfo',
+        arguments: contactsList.toSet(),
       );
-    }
-  }
-
-  void showImagesPickerAction({
-    required BuildContext context,
-  }) async {
-    if (isCreatingRoom) {
-      return;
-    }
-    if (PlatformInfos.isWeb) {
-      _getImageOnWeb(context);
-      return;
-    }
-    final currentPermissionPhotos = await getCurrentMediaPermission();
-    final currentPermissionCamera = await getCurrentCameraPermission();
-    if (currentPermissionPhotos != null && currentPermissionCamera != null) {
-      final imagePickerController = createImagePickerController();
-      groupNameFocusNode.unfocus();
-      showImagePickerBottomSheet(
-        context,
-        currentPermissionPhotos,
-        currentPermissionCamera,
-        imagePickerController,
+    } else {
+      context.push(
+        '/rooms/newprivatechat/newgroup/newgroupinfo',
+        extra: contactsList.toSet(),
```

Comment:
same

Reply:
no need, please, the logic is easy to understand enough